### ISSUE TITLE
Packaging upgrade tests fail installing 6.2.x (OSS)

### ISSUE THREAD

**Issue by tvernum** [elastic/elasticsearch#45476]
Relates: https://github.com/elastic/elasticsearch/issues/30682

**CI**: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+7.x+packaging-tests/247

Attempting to test the upgrade from 6.2.2 RPM fails at install because it is incorrectly looking for the `-oss-` flavour, which only exists from 6.3 onwards.

```
not ok 146 [UPGRADE] install old version
# (from function `install_package' in file /elasticsearch/qa/vagrant/bats/utils/packages.bash, line 86,
# in test file /elasticsearch/qa/vagrant/bats/oss/80_upgrade.bats, line 60)
# `install_package -v $(cat upgrade_from_version)' failed
# error: open of elasticsearch-oss-6.2.2.rpm failed: No such file or directory
```

**Comment by rjernst** [comment]
This was a regression from a recent refactoring of the packaging tests. I've restored the old behavior (skipping pre 6.3 upgrade versions) in https://github.com/elastic/elasticsearch/pull/45478

**Comment by rjernst** [comment]
Fixed by #45478
